### ISSUE TITLE
Confirm OTP

### ISSUE THREAD

**Issue by Yorxxx** [Yorxxx/ok-fake-server#14]
Should implement the method that validates the SMS message

## Request

    URL http://localhost:8000/api/transactions/{transaction-id}/signature_confirmation/
    Request Method:POST
    Status Code:404 Not Found
    Remote Address:127.0.0.1:8000
    Referrer Policy:no-referrer-when-downgrade

## Request Headers
    Accept:application/json, text/plain, /
    Accept-Encoding:gzip, deflate, sdch, br
    Accept-Language:es-ES,es;q=0.8
    authorization:Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOjkyNiwiaXNzIjoiaHR0cDpcL1wvbG9jYWxob3N0OjgwMDBcL2FwaVwvYXV0aGVudGljYXRlIiwiaWF0IjoxNDkxOTkxMDE0LCJleHAiOjE0OTE5OTQ2MTQsIm5iZiI6MTQ5MTk5MTAxNCwianRpIjoiYzMzZDRkMWRkZThhMDRiNWJjNWU5NTJjMWMwOTY5ZTUifQ.DtqxJm-2WesBkeT8LX3ynJw_Vfu_ZFidUroGoQw5Yek

## Request Payload

```javascript
{"optSmsCode": "foo"}
```

**Comment by Yorxxx** [comment]
According to sourcecode from client, it seems it should just return a 200 status code.

```javascript
...
var otpStatus = response.status;
if(otpStatus !== 200) {
    e.preventDefault();
    toast.loading.hide();
    toast.error('', literals.wrongOtpCode);
    return;
}
```javascript

**Comment by Yorxxx** [comment]
Implemented on #13

**Comment by Yorxxx** [comment]
Fixed on #18
